idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/kettlereboiler.py

```diff
@@ -0,0 +1,367 @@
+        super(KettleReboilerData, self).build()
+
+        # Build Control Volume for liquid Phase
+        self.liquid_phase = ControlVolume0DBlock(default={
+            "dynamic": False,
+            "property_package": self.config.liquid_side.property_package,
+            "property_package_args": self.config.liquid_side.property_package_args})
+
+        self.liquid_phase.add_state_blocks(has_phase_equilibrium=False)
+
+        # Add only momentum balance
+        # Combined  Mass and Energy balances for  Liquid and Vapor phases
+        # are added as performance equations
+
+        self.liquid_phase.add_momentum_balances(
+            balance_type=MomentumBalanceType.pressureTotal,
+            has_pressure_change=False)
+
+        # ======================================================================
+        # Build Control Volume for vapor Phase
+        self.vapor_phase = ControlVolume0DBlock(default={
+            "dynamic": False,
+            "property_package": self.config.vapor_side.property_package,
+            "property_package_args": self.config.vapor_side.property_package_args})
+
+        self.vapor_phase.add_state_blocks(has_phase_equilibrium=False)
+
+        # Energy balance not added: Exit temperature is the equilibruim Temp.
+
+        self.vapor_phase.add_material_balances(
+            balance_type=MaterialBalanceType.componentTotal,
+            has_mass_transfer=False,
+            has_phase_equilibrium=False,
+            has_rate_reactions=False)
+
+        self.vapor_phase.add_momentum_balances(
+            balance_type=MomentumBalanceType.pressureTotal,
+            has_pressure_change=False)
+
+        # ======================================================================
+        # Add Ports to control volumes
+        # Liquid Phase
+        self.add_inlet_port(name="liquid_inlet",
+                            block=self.liquid_phase, doc='inlet Port')
+        self.add_outlet_port(name="liquid_outlet",
+                             block=self.liquid_phase, doc='outlet Port')
+
+        # Vapor Phase : No inlet port
+        self.add_outlet_port(name="vapor_outlet",
+                             block=self.vapor_phase, doc='outlet Port')
+        # ======================================================================
+        # Add performace equation method
+        self._make_params()
+        self._make_performance_method()
+
+    def _make_params(self):
+
+        # Internal variables for custom mass and energy balance
+
+        self.y = Var(self.flowsheet().time,
+                     self.config.vapor_side.property_package.component_list,
+                     initialize=0.5,
+                     units=None,
+                     doc="Vapor component composition")
+
+        self.vf = Var(self.flowsheet().time,
+                      initialize=0.5,
+                      units=None,
+                      doc="Vapor phase fraction")
+
+        self.FV = Var(self.flowsheet().time,
```

Comment:
1. Again, why is this a unit level variable?
2. A general model cannot use fixed units like this.